src/pages/home/report/ReportActionsList.js

```diff
@@ -341,20 +341,11 @@ function ReportActionsList({
         }
     };
 
-<<<<<<< HEAD:src/pages/home/report/ReportActionsList.js
-    const trackVerticalScrolling = useCallback((event) => {
+    const trackVerticalScrolling = (event) => {
         scrollingVerticalOffset.current = event.nativeEvent.contentOffset.y;
         handleUnreadFloatingButton();
         onScroll(event);
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, []);
-=======
-    const trackVerticalScrolling = (event: NativeSyntheticEvent<NativeScrollEvent>) => {
```

Comment:
I think ideally we want this line instead of 344

Reply:
We reverted the changes to this file, so this is no longer a TS file. Am I missing something?

Reply:
actually sorry nvm!